netci.groovy

```diff
@@ -86,6 +86,11 @@ class Constants {
                'tailcallstress'                 : ['COMPlus_TailcallStress' : '1'],
                'jitsse2only'                    : ['COMPlus_EnableAVX' : '0', 'COMPlus_EnableSSE3_4' : '0'],
                'jitnosimd'                      : ['COMPlus_FeatureSIMD' : '0'],
+               'jitnox86hwintrinsic'            : ['COMPlus_EnableIncompleteISAClass' : '1', 'COMPlus_EnableSSE' : '0' , 'COMPlus_EnableSSE2' : '0' , 'COMPlus_EnableSSE3' : '0' , 'COMPlus_EnableSSSE3' : '0' , 'COMPlus_EnableSSE41' : '0' , 'COMPlus_EnableSSE42' : '0' , 'COMPlus_EnableAVX' : '0' , 'COMPlus_EnableAVX2' : '0' , 'COMPlus_EnableAES' : '0' , 'COMPlus_EnableBMI1' : '0' , 'COMPlus_EnableBMI2' : '0' , 'COMPlus_EnableFMA' : '0' , 'COMPlus_EnableLZCNT' : '0' , 'COMPlus_ EnablePCLMULQDQ' : '0' , 'COMPlus_EnablePOPCNT' : '0'],
```

Comment:
> 'COMPlus_EnableSSE' : '0' , 'COMPlus_EnableSSE2' : '0'

Just wondering, are these only controlling codegen for HWIntrinsics today? I know the others (where applicable) control codegen more broadly.

Reply:
> Just wondering, are these only controlling codegen for HWIntrinsics today? 

Yes, these two just control HW intrinsics, no impact for the floating-point part.

Reply:
Should there be a space in `COMPlus_ EnablePCLMULQDQ'`?

Reply:
@benaadams good catch, thank you!